ceph-extra-exporter/README.md

```diff
@@ -3,6 +3,34 @@ ceph-extra-exporter
 
 `ceph-extra-exporter` exposes extra metrics using some Ceph commands.
 
+## How to work
+
+ceph-extra-exporter gets metrics using `ceph`, `rados-gw-admin`, and related commands and export metric values like as follows.
+
+- Run commands such as `ceph` written in the code. The output is in JSON format.
+- Modify output JSON to specify format using jq filter.
+- Decode formatted JSON and export these values as the metrics.
+
+This logic is implemented [here](https://github.com/cybozu/neco-containers/blob/91eb3f4771cb0e02513e078e60c98358a88bd280/ceph-extra-exporter/executer.go#L87-L126).
+The commands to get values and arguments for jq are [hard-coded](https://github.com/cybozu/neco-containers/blob/91eb3f4771cb0e02513e078e60c98358a88bd280/ceph-extra-exporter/main.go#L18-L54), so to add metrics, please modify the program by reading [`How to add metrics`](#how-to-add-metrics) section.
+
+## How to run
+
+To use `ceph` and `radosgw-admin` commands in the docker container of ceph-extra-exporter, configuration files for ceph commands are required. To retrieve those files, deploy ceph-extra-exporter container as a sidecar of rook toolbox sharing the `/etc/ceph` directory.
```

Comment:
> To retrieve those files, deploy ceph-extra-exporter container as a sidecar of rook toolbox sharing the `/etc/ceph` directory.

This sentence is too concrete. The ceph-extra-exporter requires `ceph.conf` as you noted, but it need not be a sidecar of rook toolbox.